reflex/app.py

```diff
@@ -141,7 +141,7 @@ class App(Base):
     # The radix theme for the entire app
     theme: Optional[Component] = themes.theme(accent_color="blue")
 
-    def __init__(self, *args, **kwargs):
+    def __init__(self, exception_handler = None, *args, **kwargs):
```

Comment:
i would prefer adding the exception_handler as an attribute to `rx.App`. Since `rx.App` uses pydantic (inherits from `rx.Base` which is a pydantic `BaseModel`)  pydantic would automatically add this `__init__` arg for you